frontend/src/container/AdminAccount.jsx

```diff
@@ -120,11 +120,14 @@ export class Account extends React.Component {
     this.setState(prev => ({
       userToEditWorkspaceList: prev.userToEditWorkspaceList.map(ws => (
         ws.id === data.workspace.workspace_id
-          ? { ...ws, memberList: ws.memberList.map(member => (
+          ? {
+            ...ws,
+            memberList: ws.memberList.map(member => (
               member.id === Number(state.userToEditId)
                 ? { ...member, doNotify: data.member.do_notify }
                 : member
-          ))}
+            ))
+          }
```

Comment:
Ah ah, I hesitated to suggest you this indentation and told myself "no, this is too strict".
This looks good.
:+1: :+1: